src/plugins/tooltip/c-tooltip.vue

```diff
@@ -33,9 +38,7 @@
   } from 'vue';
   import { createPopper, Instance, Options } from '@popperjs/core';
```

Comment:
suggestion: In my recent Vue3 upgrade, we switched from `popper.js` to `floating-vue`. Not sure if it's project-specific, but it is maybe worth to discuss it.

Reply:
@BeneRichi good input. Can you create an additional ticket to replace it with information where to find the related code. Not sure where you added this.

Reply:
I have opened an issue: https://github.com/valantic/vue-template/issues/354.